scripts/android/cmake.sh

```diff
@@ -156,7 +156,7 @@ fi
 
 print Y "Compiling apk_install_target in .tmp/"
 # This compiles the client and generates a mozillavpn.so
-cmake --build .tmp -j$JOBS
+cmake --build .tmp --target mozillavpn -j$JOBS
```

Comment:
That's not the target you want - that just builds libmozillavpn.so, on android qt add's other default targets that will also generate the gradle project, hence why the builds are not working  
keep the default, just flag the tests off :)